### DISCUSSION TITLE
Distinct role name per client

### DISCUSSION THREAD

**Discussion by razi-ahmad** [sventorben/keycloak-restrict-client-auth#286] [Ideas]
Hi, I want to use different role name per client. I didn't see in the code, different role name can be configured from the UI.

**Comment by sventorben** [comment]
What is the reasoning behind that?

**Reply by razi-ahmad** [reply]
E.g UserA can access App1(ClientA) and other userB can access App2(clientB).  How it will be possible to control per role per client.

**Reply by sventorben** [reply]
Please check this thread and let me know if the solution would work for you.

https://github.com/sventorben/keycloak-restrict-client-auth/issues/118#issuecomment-1300013191
